src/wiki/plugins/inputs/models.py

```diff
@@ -0,0 +1,33 @@
+from __future__ import absolute_import, unicode_literals
+
+from django.db import models
+from django.utils.encoding import python_2_unicode_compatible
+from django.utils.translation import ugettext_lazy as _
+from wiki.models.pluginbase import ArticlePlugin
+from wiki.core import compat
+
+
+@python_2_unicode_compatible
+class Input(ArticlePlugin):
+    owner = models.ForeignKey(
+        compat.USER_MODEL, verbose_name=_('owner'),
+        null=True, related_name='owned_inputs',
+        help_text=_('The author of the input. The owner always has both read access.'),
+        on_delete=models.SET_NULL)
+
+    key = models.CharField(max_length=28)
+    val = models.TextField()
```

Comment:
I'm not sure if I would really agree that this is a very structured way to organize data. I added this manifesto to guide what I think this project should do to keep its internal architecture nice and to have deployments and plugin ecosystems have the same over-all goals:

https://github.com/django-wiki/django-wiki/blob/master/CONTRIBUTING.md#manifesto